engine/src/test/java/org/camunda/bpm/engine/test/api/repository/ProcessDefinitionQueryTest.java

```diff
@@ -855,7 +865,19 @@ public void testQueryOrderByVersionTag() {
 
   @Test
   public void testQueryByStartableInTasklist() {
-    assertThat(repositoryService.createProcessDefinitionQuery().startableInTasklist().count()).isEqualTo(4);
+    // given
+    // three definitions with startableInTasklist=true
+    // one definition with startableInTasklist=true
```

Comment:
🙃 There are three processes having startableInTasklist=`false` from `one.bpmn20.xml`, `two.bpmn20.xml` and one with `false` from `three_.bpmn20.xml`.
```suggestion
    // one definition with startableInTasklist=false
```

Reply:
Yes, that's correct. The `startableInTaskList` property is the one under test here. So far, all models had a value of `true`, but we never tested `false`. This is why I added it to the last remaining model.

Reply:
Ah, sorry. Now I read your comment more closely. The comment is wrong. :D